.github/workflows/comment-labeled.yml

```diff
@@ -1,29 +1,35 @@
-name: Comment on issues and PRs when labelled
+name: Comment on issues and PRs when labeled
```

Comment:
I don't remember if we prefer American or British English? https://www.grammarly.com/blog/labeled-labelled/

Reply:
It's more consistent with github's "labeled".